src/style.css

```diff
@@ -0,0 +1,72 @@
+.searchBar {
+  position: relative;
+  margin-top: 20px;
+}
+
+.inputSearchBar {
+  width: 100%;
+  height: 70px;
+  padding: 0 0 0 62px;
+  box-sizing: border-box;
+  outline: none;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  font-weight: 900;
```

Comment:
create separate class for big search bar

Reply:
thanks, fixed)